imports/ui/components/pageheader/pageheader.html

```diff
@@ -1,12 +1,17 @@
 <template name="pageheader">
-	<div class="clearfix">
```

Comment:
This is intended to be a component template that can be used on multiple pages. Right now it has only been tested on the profile page, but it should not be difficult to tweak to get working on other pages